### DISCUSSION TITLE
loading data from +page.js despite +layout.server.js data

### DISCUSSION THREAD

**Discussion by Graciasjr** [sveltejs/kit#10058] [Q&A] [answered]
Hello everybody. 
Please , how to load data from my +page.js when my +layout.server.js already returns data to child pages ?

**Comment by aloker** [comment, marked as answer]
The data coming from the load functions of all layouts and the page is merged together to form the data prop passed to the page. That is, the data prop will have all the properties of the results of the +layout.server.js as well as the properties of the result of +page.js (equally named props in the page's load result overwrite those from the layout's load function)

**Reply by Graciasjr** [reply]
Ok if I understand correctly, I could then load both the data from layout.server.js and the data from my page.js provided that the props which are returned are not the same

**Reply by Graciasjr** [reply]
yes it is verified. 
Thanks so Much !
